constants.py

```diff
@@ -0,0 +1,6 @@
+"""
+A collection of app constants.
+"""
+
+BOARD_SIZE = 5
+BOARD_MAXCOLS = BOARD_SIZE * 2 - 1
```

Comment:
I usually centralize config in a `.py` like this so I can just do `from constants import BOARD_SIZE` etc. wherever else in the program I need them